crates/sui-adapter-transactional-tests/tests/checkpoint/clock.move

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//# init
+
+//# checkpoint-prologue --timestamp-ms 43
+
+//# view-object 102
+
+//# checkpoint-prologue --timestamp-ms 45
```

Comment:
Probably fine for what you are using it for, but just keep in mind the `sui-adapter-transactional-tests` do _not_ go through the authority code paths. I've been thinking for a while that it should, but would be a major overhaul. 

Would be nice if we could then use the transaction builder code here and really have some easy to read end to end tests

Reply:
Yeah, I added a separate test for the authority flows -- this test is just to make sure that the checkpoint prologue transaction does what we expect it to, to on-chain objects when it runs.